From: Xichao Zhao <zhao.xichao@vivo.com>
Subject: lib/fault-inject-usercopy.c: use PTR_ERR_OR_ZERO() to simplify code
Date: Tue, 12 Aug 2025 16:40:45 +0800

Use the standard error pointer macro to shorten the code and simplify.

Link: https://lkml.kernel.org/r/20250812084045.64218-1-zhao.xichao@vivo.com
Signed-off-by: Xichao Zhao <zhao.xichao@vivo.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/fault-inject-usercopy.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/lib/fault-inject-usercopy.c~lib-use-ptr_err_or_zero-to-simplify-code
+++ a/lib/fault-inject-usercopy.c
@@ -22,10 +22,8 @@ static int __init fail_usercopy_debugfs(
 
 	dir = fault_create_debugfs_attr("fail_usercopy", NULL,
 					&fail_usercopy.attr);
-	if (IS_ERR(dir))
-		return PTR_ERR(dir);
 
-	return 0;
+	return PTR_ERR_OR_ZERO(dir);
 }
 
 late_initcall(fail_usercopy_debugfs);
_
